assets/src/edit-story/components/inspector/document/publish/index.js

```diff
@@ -93,7 +93,7 @@ const StyledToggleIcon = styled(ToggleIcon)`
 
 function PublishPanel() {
   const {
-    state: { users },
+    state: { users, isUsersLoading },
```

Comment:
Where is `isUsersLoading` setup and set?

Reply:
@spacedmonkey This logic is on `InspectorProvider` where getAllUsers function called.

Reply:
Fair enough.